install/linux/usr/share/odemis/sim/sparc2-streakcam-sim.odm.yaml

```diff
@@ -0,0 +1,446 @@
+    properties: {
+        scale: [8, 8], # (ratio) : start with a pretty fast scan
+        dwellTime: 10.e-6, # s
+        magnification: 100, # (ratio)
+    },
+    affects: ["SEM Detector", "Camera"] # affects the CCD in case of cathodoluminescence
+}
+
+# Must be connected on AI 0/AI GND
+"SEM Detector": { # aka ETD
+    # Internal child of SEM Scan Interface, so no class
+    role: se-detector,
+    init: {
+        channel: 0, # 0-> sawtooth waves, 1-> square waves
+        limits: [-3, 3] # V
+    },
+}
+
+# CL PMT: composed of the control unit + the PMT output connected to the DAQ board
+
+# Must be connected on AI 1/AI GND
+"CL PMT": { # the raw detector of the PMT
+    # Internal child of SEM Scan Interface, so no class but instead a creator
+    creator: "SEM Scan Interface",
+    role: null,
+    init: {
+        channel: 1, # 0-> sawtooth waves, 1-> square waves
+        limits: [3, 0], # V, with contrast reversed
+    },
+}
+
+"CL PMT control unit": {
+    class: pmtctrl.PMTControl,
+    role: null,
+    power_supplier: "Power Control Unit",
+    init: {
+        port: "/dev/fake", # for simulator
+        prot_time: 0.0002,  # s
+        prot_curr: 50.e-6,  # A
+    }
+}
+
+"CL Detector": {
+    class: pmtctrl.PMT,
+    role: cl-detector,
+    init: {
+        settle_time: 0.1, # s, time to wait before detector is ready
+    },
+    children: {
+       "detector": "CL PMT",
+       "pmt-control": "CL PMT control unit",
+    },
+}
+
+# In reality, this is a Zyla, but you need libandor3-dev to simulate an AndorCam3
+# Depending exactly on the configuration, it might also be used for spectrometer
+"Camera": {
+    class: andorcam2.AndorCam2,
+    role: ccd,
+    power_supplier: "Power Control Unit",
+    init: {
+       device: "fake",
+       image: "sparc-ar-mirror-align.h5", # only for simulator
+    },
+}
+
+"Spectrometer Vis-NIR": {
+    class: spectrometer.CompositedSpectrometer,
+    role: spectrometer,
+    children: {detector: "Camera", spectrograph: "Spectrograph"},
+    init: {
+        transp: [1, -2], # only applied to the spectrometer data (not raw CCD)
+    },
+}
+
+# simple simulated streak camera
+"Streak Camera": {
+    class: simstreakcam.StreakCamera,
+    role: streak-cam,
+    init: {
+    },
+    children: {readoutcam: "Streak Readout CCD",
+               delaybox: "Streak Delay Generator",
+               streakunit: "Streak Unit",
+               spectrograph: "Spectrograph"},
+}
+
+# for testing with real HW
+#"Streak Camera": {
+#    class: hamamatsurx.StreakCamera,
+#    role: streak-cam,
+#    init: {
+#        host: "172.16.4.2",  # Braunschweig System
+#        port: 1001
+#    },
+#    children: {readoutcam: "Streak Readout CCD",
+#               delaybox: "Streak Delay Generator",
+#               streakunit: "Streak Unit",
+#               spectrograph: "Spectrograph"},
+#}
+
+"Streak Readout CCD": {
+    role: streak-ccd,
+    init: {
+        image: "sparc-tempSpec-sim.h5",  # only for simulator!
```

Comment:
Either use an image file which already exists, or add a _small_ one (eg, 2 frames) with "nice"/typical content.

Reply:
I don't have one yet, so I added a not so nice one with just background noise, but would like to replace it when we get actual data. Or do you prefer adding one when we have it and change the yaml file accordingly?